### DISCUSSION TITLE
Calculating cohen's d for two columns differing in length

### DISCUSSION THREAD

**Discussion by Dallak** [easystats/effectsize#467] [Q&A] [answered]
**Question and context**
Dear all,
Is it possible to use this great package for calculating cohen's d with CI for two columns differing in length?
Thanks in advance!

**Comment by mattansb** [comment, marked as answer]
Not a problem

``` r
x <- 1:10 # length = 10
y <- 7:20 # length = 14

effectsize::cohens_d(x, y)
#> Cohen's d |         95% CI
#> --------------------------
#> -2.13     | [-3.14, -1.09]
#> 
#> - Estimated using pooled SD.
```

<sup>Created on 2022-08-14 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>
